lib/node/runner/node_runner.go

```diff
@@ -246,6 +248,15 @@ func (nr *NodeRunner) Ready() {
 		TransactionsHandler,
 	).Methods("GET", "POST").MatcherFunc(common.PostAndJSONMatcher)
 
+	// pprof
+	if DebugPProf == true {
+		nr.network.AddHandler("/debug/pprof/cmdline", pprof.Cmdline)
```

Comment:
According to the doc, this is done automatically by the package init ?

Reply:
Yes but we use gorilla mux. The binding in http/pprof doesn’t works. :-> 
- https://stackoverflow.com/questions/19591065/profiling-go-web-application-built-with-gorillas-mux-with-net-http-pprof